### DISCUSSION TITLE
How can I convert <Text> tag to <Path> tag

### DISCUSSION THREAD

**Discussion by jacksonwj** [vvvv/SVG#985] [Q&A]
## Description
There is `<Text>` tag (or other tag), I wanna convert it to `<Path>` tag, such as
```
<path id="1" fill="#000000" d="M1...." />
```

When convert to `<Path>`, the svg file will be opened in CorelDRAW (or other tool).

How can I do, help please.

## Example data
```
<text x="100" transform="scale(1, 0.8)" font-size="50" font-weight="bold" writing-mode="tb">Demo Content</text>
```

## Used Versions
3.4.2

**Comment by H1Gdev** [comment]
I don't know of such a conversion tool.

I consider the following method.
`GraphicPath` is generated by [Path method](https://github.com/svg-net/SVG/blob/090a480c4ba3b073bb8a238a0d063212ff1ccef7/Source/Text/SvgTextBase.Drawing.cs#L82) of Text, and [Paths](https://github.com/svg-net/SVG/tree/v3.4.2/Source/Paths) is created from it.
